worlds/witness/hints.py

```diff
@@ -168,6 +170,50 @@ def get_priority_hint_locations(world: "WitnessWorld") -> List[str]:
     return priority
 
 
+def try_getting_location_group_for_location(world: "WitnessWorld", hint_loc: Location) -> Tuple[str, str]:
+    allow_regions = world.options.vague_hints == "experimental"
+
+    possible_location_groups = {
+        k: v for k, v in world.multiworld.worlds[hint_loc.player].location_name_groups.items()
+        if hint_loc.name in v
+    }
+
+    locations_in_that_world = {
+        location.name for location in world.multiworld.get_locations(hint_loc.player) if location.address is not None
+    }
```

Comment:
```suggestion
    locations_in_that_world = {
        location.name for location in world.multiworld.get_locations(hint_loc.player) if location.address
    }
```
Not sure if the `not None` is for clarity or not

Reply:
I'm not sure, games could potentially start using location id 0 as of 1933 being merged I think? Or maybe that's reserved. Either way, `is not None` is the most accurate way to check for event locations

Reply:
According to Berserker, no, games can still not use address 0. Either way, I think I'd like to keep this

Reply:
Wait, I'm a dumbass. We have location.is_event now